tests/jeos/kdump_disabled.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify Kdump is (by default) not enabled in JeOS image
```

Comment:
Why is that important? Do you have a fate/bugzilla/progress ticket about that which you could reference in the third line after "# Maintainer: …" with e.g. "Tags: #1234"?

Reply:
It's plainly stated in Product Requirements Document to JeOS as "kdump disabled". I haven't found any addressable resource for that as well as no other information to this regard...

But, oddly enough we have the test already in console tests, me just not knowing about this... Removed it and kept just the snapshotting part.